drivers/clock_control/clock_stm32_ll_common.c

```diff
@@ -75,7 +75,8 @@
  */
 static void config_bus_clk_init(LL_UTILS_ClkInitTypeDef *clk_init)
 {
-#if defined(CONFIG_SOC_SERIES_STM32WBX) || defined(CONFIG_SOC_SERIES_STM32WLX)
+#if defined(CONFIG_SOC_SERIES_STM32WBX) || \
+	defined(CONFIG_SOC_SERIES_STM32WLX) && defined(DUAL_CORE)
```

Comment:
To avoid ambiguity, I would add some parenthesis
```suggestion
#if defined(CONFIG_SOC_SERIES_STM32WBX) || \
	(defined(CONFIG_SOC_SERIES_STM32WLX) && defined(DUAL_CORE))
```